src/javascript/app_2/Constants/ui.js

```diff
@@ -1,2 +1,20 @@
 export const MAX_MOBILE_WIDTH = 767;
 export const MAX_TABLET_WIDTH = 980;
+
+export const COLOR_CODES = {
+    WHITE       : '#fff',
+    LIGHT_GRAY  : '#fafafb',
+    GRAY        : '#e9e9ed',
+    DARK_WHITE  : '#ccc',
+    LIGHT_GRAY_2: '#b0b3bf',
+    YELLOW      : '#fef1cf',
+    ORANGE      : '#e98024',
+    LIGHT_RED   : '#f44336',
+    LIGHT_GREEN : '#4caf50',
+    GREEN       : '#2e8836',
+    LIGHT_BLUE  : '#1565c0',
+    BLUE        : '#2a3052',
+    DARK_BLUE   : '#15212d',
+    LIGHT_BLACK : '#333',
+    BLACK       : '#000',
+};
```

Comment:
If these colors are just for icons, then I believe I found a better solution [in my PR](https://github.com/binary-com/binary-static/pull/4169). There we leave the styling to CSS and color constants are only in CSS. Here we would need to keep both sass constants and COLOR_CODES updated and in sync.

Reply:
In my PR CSS also doesn't need to know icon's inner structure.

Reply:
Sure, thanks.